File: 151002/Bybikov/FirstRestSolution/src/main/java/com/example/rv/impl/issue/Issue.java
package com.example.rv.impl.issue;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Builder
public class Issue {
    Long id;
    Long creatorId;
    String title;
    String content;
    String created;
    String modified;
}

File: 151001/Hlebanova/LR2/LR2/Models/Issue.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;

namespace LR2.Models;

[Table("tbl_issues")]
[Index(nameof(Title), IsUnique = true)]
public class Issue
{
    public long Id { get; set; }

    [ForeignKey("Creator")] public long? CreatorId { get; set; }

    [MaxLength(32)] public string Title { get; set; } = string.Empty;
    [MaxLength(2048)] public string Content { get; set; } = string.Empty;

    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public DateTime? Created { get; init; }

    [DatabaseGenerated(DatabaseGeneratedOption.Computed)]
    public DateTime? Modified { get; set; }

    public Creator? Creator { get; set; }
    public List<Comment> Posts { get; } = [];
    public List<Label> Labels { get; } = [];
}